model_analyzer/analyzer.py

```diff
@@ -127,18 +128,20 @@ def analyze(self, mode, verbose):
             config=self._config,
             gpu_info=gpu_info,
             result_manager=self._result_manager)
+        self._result_table_manager = ResultTableManager(self._config,
+                                                        self._result_manager)
 
         # Create result tables, put top results and get stats
-        self._result_manager.create_tables()
         self._result_manager.compile_and_sort_results()
         self._report_manager.create_summaries()
         self._report_manager.export_summaries()
 
         # Dump to tables and write to disk
-        self._result_manager.tabulate_results()
-        self._result_manager.export_results()
+        self._result_table_manager.create_tables()
+        self._result_table_manager.tabulate_results()
+        self._result_table_manager.export_results()
```

Comment:
Is there a reason why these are called separately? (e.g. cases where we create tables but not tabulate results). Maybe it would be cleaner to hide these steps inside the ResultTableManager.